protected/humhub/modules/stream/widgets/WallStreamFilterNavigation.php

```diff
@@ -59,15 +64,12 @@
 class WallStreamFilterNavigation extends FilterNavigation
 {
     /**
-     * Left panel position
+     * Panel columns
      */
-    const PANEL_POSITION_LEFT = 0;
-
-    /**
-     * Left panel position
-     */
-    const PANEL_POSITION_CENTER = 1;
-    const PANEL_POSITION_RIGHT = 2;
+    const PANEL_COLUMN_1= 0;
```

Comment:
@s-tyshchenko I am a bit concerned that modules may use these constants.

Can you please check if modules from https://github.com/humhub or https://github.com/humhub-contrib access these constants?

If they are not used, however, we need to mark that as a Breaking Change in the doc.

Reply:
No, I haven't found any usage of these constants.